docs/self-managed/modeler/web-modeler/configuration.md

```diff
@@ -30,7 +30,52 @@ Web Modeler requires a PostgreSQL database as persistent data storage (other dat
 | `RESTAPI_DB_USER`     | Database user name     | `modeler-user`         |
 | `RESTAPI_DB_PASSWORD` | Database user password | \*\*\*                 |
 
-#### SMTP / email
+#### Configuring SSL for the database connection
+
+The generic way to configure an SSL connection between Web Modeler and the database is as follows:
+
+- Modify database connection URL `SPRING_DATASOURCE_URL` and customize connection parameters.
```

Comment:
As a reader you could be surprised about `SPRING_DATASOURCE_URL`, because in the table above we only mention `RESTAPI_DB_HOST`, `RESTAPI_DB_PORT`, and `RESTAPI_DB_NAME`. Maybe we should add `SPRING_DATASOURCE_URL` to the table as well? Maybe something like this:

| Environment variable  | Description            | Example value          |
| --------------------- | ---------------------- | ---------------------- |
| `SPRING_DATASOURCE_URL`     | [optional]<br/>Can be used to provide a customized connection string (for example, to enable SSL-secured connections). If set, `RESTAPI_DB_HOST`, `RESTAPI_DB_PORT`, and `RESTAPI_DB_NAME` don't need to be provided. | `jdbc:postgresql://postgres.example.com:5432/modeler-db?sslmode=require` |